packages/fxa-settings/src/components/Settings/Modal/index.tsx

```diff
@@ -68,69 +68,66 @@ export const Modal = ({
           )}
           ref={modalInsideRef}
         >
-          <div
-            tabIndex={0}
-            ref={tabFenceRef}
-            data-testid="modal-tab-fence"
-            className="outline-none"
-          />
-          <div className="flex justify-end pe-2 py-2">
-            <button
-              data-testid="modal-dismiss"
-              onClick={(event) => onDismiss()}
-              title={ftlMsgResolver.getMsg('modal-close-title', 'Close')}
+          <div tabIndex={0} ref={tabFenceRef} data-testid="modal-tab-fence">
+            <div className="flex justify-end pe-2 py-2">
+              <button
+                className="border-2 border-transparent"
+                data-testid="modal-dismiss"
+                onClick={(event) => onDismiss()}
+                title={ftlMsgResolver.getMsg('modal-close-title', 'Close')}
+              >
+                <CloseIcon className="w-4 h-4 m-3 fill-current" />
+              </button>
+            </div>
+
+            <div
+              aria-describedby={descId}
+              aria-labelledby={headerId}
+              className="px-6 tablet:px-10 pb-10"
+              data-testid="modal-information"
+              role="dialog"
             >
-              <CloseIcon className="w-4 h-4 m-3" role="img" />
-            </button>
-          </div>
+              {children}
+
+              {hasButtons && (
+                <div className="flex justify-center mx-auto mt-6 max-w-64">
+                  {hasCancelButton && (
+                    <FtlMsg id="modal-cancel-button">
+                      <button
+                        className="cta-neutral mx-2 flex-1 cta-base-p"
+                        data-testid="modal-cancel"
+                        onClick={(event) => onDismiss()}
+                      >
+                        Cancel
```

Comment:
Does this need to be localized, or does the `<FtlMsg>` wrapper do that for us here?